backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQHealthCheckTest.java

```diff
@@ -61,6 +61,7 @@ void checkShouldReturnUnhealthyWhenRabbitMQIsNotRunning(DockerRabbitMQ rabbitMQ)
     }
 
     @Test
+    @Disabled("connection don't recover instantly, we should try several time (depending on heartbeat rabbit conf")
```

Comment:
Then we might need a retry logic with a timeout?

Reply:
yes